packages/components/buttons/secondary-button/src/secondary-button.tsx

```diff
@@ -65,8 +66,13 @@ export type TSecondaryButtonProps<
   size?: 'medium' | 'big';
   /**
    * Indicates the color scheme of the button.
+   * @deprecated Use `tone` instead.
    */
   theme?: 'default' | 'info';
+  /**
+   * Indicates the tone of the button.
+   */
+  tone?: 'default' | 'info';
```

Comment:
I think we should called the first option **secondary** as we don't have a _default_ value anywhere else for a **tone** property.

Reply:
Updated [2bf037d](https://github.com/commercetools/ui-kit/pull/2576/commits/2bf037dbac36c1f6e460599217d4a95a9b087cd0)